### PR TITLE
318 update lobby status endpoint game visualization flag

### PR DESCRIPTION
**Pull request by flamion**

### CODE REVIEW

**Comment by flamion** [comment]
blocked until #323 is merged as the architecture diagram still has to be updated and I **DO NOT** want to merge the draw.io architecture diagram by hand :sob:

**Review by nikkischnelle — APPROVED** [review]
lgtm

**Pull request merged**
